lib/bitmap-transform.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function invertColors(obj) {
+  let bufferArray = new Int32Array(obj);
+
+  let invertTransform = bufferArray.reduce(function(acc, ele, index) {
+    if (index < 54) acc.push(ele);
+    if (index >= 54 && index < 1078) {
+      let inverted = 255 - ele;
+      acc.push(inverted);
+    }
+    if (index >= 1078) acc.push(ele);
+    return acc;
+  }, []);
+
+  let newBuffer = Buffer.from(invertTransform);
+
+
+  let invertFileHelper = function(obj) {
```

Comment:
@khalidM3 @reganoneill @dkulp23 `invertFileHelper` looks good but could be more dynamic - there's a few areas in your application where you are reading/writing files - best approach would be to have a single `read-file-helper` that is encapsulated for use in other files - this way you can test it with ease and dynamically pass the filename to it